module/VuFind/src/VuFind/Db/Service/CommentsService.php

```diff
@@ -65,7 +65,7 @@ public function createEntity(): Comments
     public function getForResource(string $id, $source = DEFAULT_SEARCH_BACKEND)
     {
         $resource = $this->getDbService(\VuFind\Db\Service\ResourceService::class)
-            ->findResource($id, $source);
+            ->findResource($id, $source, false);
```

Comment:
I think you are almost certainly correct to make this false here and in other getters, but I'm curious what led you to make this change. Was it to address a specific problem, or just an improvement you decided to make along the way? (I'm just curious in case it was revealed by some other issue that it might be good for me to know about, or if you're just thinking ahead to optimization). Either way, thanks! :-)

Reply:
When the method was partially migrated, it did not have the functionality of creating a new record if it doesn't exist, so $create parameter that defaults to True was not passed. The logic in dev branch, is migrated as is, so I had pass the parameter to override the default action of creating a record if doesn't exist.